### PR TITLE
Updated README to remove reference to retries.

### PR DESCRIPTION
**Pull request by kylebgorman**
Part of #360.

Also reflowed the entire file using:

    pandoc -f markdown -t markdown --columns=80

- [x] Updated `Unreleased` in `CHANGELOG.md` to reflect the changes in code or data.

### CODE REVIEW

File: data/src/README.md [hunk 1]
```diff
@@ -48,9 +47,9 @@ Steps used to update the dataset
...
 --------------------------------------------------
 
-By default, if [`scrape.py`](scrape.py) cannot successfully complete a scrape of
-an entire language in 10 retries, it will log the language, remove the
-incomplete data scraped from that language, and move on to the next language in
+By default, if [`scrape.py`](scrape.py) cannot complete the scrape of a
+language, it will log the language, remove the incomplete data scraped from that
+language, and move on to the next language in
 [`languages.json`](languages.json). There may therefore be a few languages that
 you need to run again when the big scrape finishes. These are the steps to
```
Comment (lfashby) [inline data/src/README.md:54]: When I added the restart functionality I removed the logging of incomplete scrapes and the deletion of incomplete data from `src/scrape.py`. Other than a user ctrl + c-ing or the user's internet dropping (or Wiktionary itself dropping off the internet somehow) there isn't any reason `src/scrape.py` shouldn't complete a scrape.

I think this section can just begin with: "These are the steps to follow should you need to run the big scrape scripts for a smaller set of languages:"

**Review by lfashby — COMMENTED** [review]
Just a small comment about the new behavior of `src/scrape.py` now that we have the restart functionality in `wikipron/scrape.py`.

File: data/src/README.md (hunk 1 above)
Comment (kylebgorman) [reply data/src/README.md]: Makes sense, done.

**Review by lfashby — APPROVED** [review]
LGTM, thanks for this.

**Pull request merged**